components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/org/wso2/carbon/identity/application/mgt/ui/i18n/Resources.properties

```diff
@@ -273,5 +273,7 @@ alert.error.default.seq.name.invalid=Provided default authentication sequence na
 enter.service.provider.name.pattern=Enter service provider name pattern (* for all)
 service.provider.search=Search
 config.application.JWKS = JWKS URI
+enter.service.provider.name.pattern=Enter service provider name pattern (* for all)
```

Comment:
Aren't these properties already defined?

Reply:
remove the unnecessary line